sympy/physics/mechanics/models.py

```diff
@@ -197,6 +197,7 @@ def n_link_pendulum_on_cart(n=1, cart_force=True, joint_torques=False):
         frames.append(Bi)
 
         Pi = points[-1].locatenew('P{}'.format(i + 1), l[i] * Bi.y)
+        points[-1].set_vel(Bi, 0)
```

Comment:
Seeing that we have to add this now, I'm not sure this is the solution we want. I thought that we only want to through an error if the velocity of the fixed point is such that it has a value and that value indicates it isn't fixed. So we should allow the velocity to 1) be undefined or 2) fixed in the shared frame. Having to set this velocity to zero is just going to be annoying for users.